### ISSUE TITLE
Docker deployment doesn't expose static directory

### ISSUE THREAD

**Issue by bbassac** [danielbayerlein/dashboard#100]
When runnning dashboard with "npm run dev" , all pngs in "static" are reachable from widgets under "/static/xxxx.png".

But when running from Docker, they are not reachable (even the famous favicon).

Adding a mounting point and copying files doesn't seems enought...

**Comment by danielbayerlein** [comment]
Thanks for reporting the error! I will fix the error as soon as possible!

**Comment by bbassac** [comment]
Sorry, but it doesn't fix.
 favicon and images located in static directory are not rendered in docker mode (but works on npm run dev mode

**Comment by danielbayerlein** [comment]
@bbassac Did you rebuild the docker image after the fix (8dfda38793a5b9dffbbddb72ab9ae91ef8e860d7)?

**Comment by bbassac** [comment]
Yes i did. I temporary bypassed the problem by hosting my images on
internet.

Does it need an exposed directory in docker ?

Le mer. 13 juin 2018 à 11:18, Daniel Bayerlein <notifications@github.com> a
écrit :
